Example/Messaging/Tests/FIRMessagingServiceTest.m

```diff
@@ -336,4 +368,42 @@ - (void)testFIRMessagingSDKLocaleInFIRMessagingService {
   }
 }
 
+
+- (void)testSubscribeFailedWithInvalidToken {
+  // Mock get token is failed with FIRMessagingErrorUnknown error.
+  XCTestExpectation *subscriptionCompletionExpectation =
+      [self expectationWithDescription:@"Subscription is complete"];
+  OCMStub([_mockInstanceID
```

Comment:
Nit: You may want to use `OCMExpect` instead of `OCMStub` and add `OCMockVerifyAll(_mockInstanceID)` if invoking this method is required.

Reply:
Sure, SGTM